internal/codegen/tpl_stencil.go

```diff
@@ -138,6 +138,9 @@ func (s *TplStencil) ApplyTemplate(name string, dataSli ...interface{}) (string,
 	var data interface{}
 	if len(dataSli) == 1 {
 		data = dataSli[0]
+	} else {
+		// If no data was passed, pass through the values of the parent template
+		data = s.t.args
```

Comment:
Is there a demonstration that uses this functionality in a test?

Reply:
There is not, I'll add one!